pkgs/development/tools/rust/cargo-pgrx/0_9_7.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchCrate, rustPlatform, pkg-config, openssl, Security }:
+
+let
+  pname = "cargo-pgrx";
+  version = "0.9.7";
+in
+rustPlatform.buildRustPackage rec {
```

Comment:
instead of copying a file, is it possible to just override `src` and `version` ? ( and perhaps `checkFlags`)

Reply:
took some fiddling with overriding the `cargoDeps`, but got it working :tada: